qubeswhonix/__init__.py

```diff
@@ -112,43 +186,64 @@ def on_whonix_ws_feature_set(
         value,
         oldvalue=None,
     ):
-        '''Set NetVM and DispVM appropriately on VMs that are newly set as a
-        Whonix-Workstation VM (mostly useful for configuring StandaloneVMs
-        '''
+        """Set NetVM and DispVM appropriately on VMs that are newly set as a
+        Whonix-Workstation VM
+        """
         # pylint: disable=unused-argument,too-many-positional-arguments
-        if value == '1':
-            self.set_ws_netvm(vm.app, vm)
-            self.set_ws_dispvm(vm.app, vm)
+        if not value:
+            return
```

Comment:
We shouldn't accept values other than `1` here. We may have users who are relying on non-`1` values to say "no, this should not be treated as a Whonix-Workstation qube". Changing the semantics here breaks the API.

Reply:
Valid point. I guess the question is then whether this should be considered a documentation bug, or a code bug, and whether some danger could surface from a VM being considered a Whonix-Workstation when it wasn't previously without user interaction. If a Whonix-Workstation qube stopped being considered a Whonix-Workstation, that could open the door for identity leaks, but if a non-Whonix-Workstation qube starts behaving as a Whonix-Workstation, that might be confusing and result in some configuration getting mangled, but it probably won't endanger the user.

@adrelanos Thoughts? If you're fine with the API technically breaking here, then I think this is good.